app/assets/config/manifest.js

```diff
@@ -0,0 +1,28 @@
+//= link_tree ../images
+//= link_directory ../javascripts .js
+//= link_directory ../stylesheets .css
```

Comment:
Seems to include the error bars on `/statistics`, which I think are via https://github.com/mysociety/alaveteli/blob/develop/vendor/assets/javascripts/jquery.flot.errorbars.min.js, so I guess not

Reply:
No doesn't look like we do, I think because they are in the [asset load path](https://guides.rubyonrails.org/asset_pipeline.html#asset-organization) by default.